src/weather-bar.ts

```diff
@@ -74,11 +88,35 @@ export class WeatherBar extends LitElement {
           </div>
         `);
       }
+
+      if (this.num_empty_segments_trailing > 0) {
+        const barStylesEmpty: Readonly<StyleInfo> = { gridColumnStart: String(gridStart), gridColumnEnd: String(gridStart += this.num_empty_segments_trailing * 2) };
+        conditionBars.push(html`
+          <div class="empty" style=${styleMap(barStylesEmpty)} data-tippy-content=${emptyLabel}></div>
+        `);
+      }
     }
 
     const windCfg = this.show_wind ?? '';
     const barBlocks: TemplateResult[] = [];
     let lastDate: string | null = null;
+
+    for (let i = 0; i < this.num_empty_segments_leading; i += 1) {
```

Comment:
Consider extracting this into a small function so you can reuse this for the trailing space too without repeating yourself.